### PR TITLE
#235: AsciiText

### PR DESCRIPTION
**Pull request by ixmanuel**
See #235 

```java
String ascii = new AsciiText("Ç©mm©ñ").asString(); // Common
```

It is a requirement for SluggedText.

### CODE REVIEW

**Comment by 0crat** [comment]
@ixmanuel I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

**Comment by fabriciofx** [comment]
@ixmanuel Are you running `mvn clean install -Pqulice` before submit the PR? If not, could you do it? Thanks!

File: src/main/java/org/cactoos/text/AsciiText.java [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+
+    /**
+     * Removed non-ascii characters.
+     *
+     * @param Transliterated string.
+     * @return String without non-ascii characters.
+     * @see http://www.asciitable.com
+     */
+    private String removedNonAscii(final String value) {
+        return value.replaceAll("/(?u)[^\\x20-\\x7e]/", "");
```
Comment (alex-semenyuk) [inline src/main/java/org/cactoos/text/AsciiText.java]: @ixmanuel There is ReplacedText.

File: src/main/java/org/cactoos/text/AsciiText.java [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+        this.map = new HashMap<>(0);
+    }
+
+    @Override
+    public String asString() throws IOException {
+        String value = this.text.asString();
+        for (final Map.Entry<String, String[]> entry : 
+            this.replacements().entrySet()) {
+            for (final String unicode : entry.getValue()) {
+                value = value.replace(unicode, entry.getKey());
```
Comment (alex-semenyuk) [inline src/main/java/org/cactoos/text/AsciiText.java]: @ixmanuel There is ReplacedText.

Comment (memoyil) [inline src/main/java/org/cactoos/text/AsciiText.java]: @ixmanuel You can use ReplacedText internally

File: src/main/java/org/cactoos/text/AsciiText.java (hunk 1 above)
Comment (ixmanuel) [inline src/main/java/org/cactoos/text/AsciiText.java]: ok.

File: src/main/java/org/cactoos/text/AsciiText.java (hunk 2 above)
Comment (ixmanuel) [inline src/main/java/org/cactoos/text/AsciiText.java]: ok.

**Comment by ixmanuel** [comment]
In windows (appveyor) can not manage 5 unicode characters. What do you recommend?

- [ERROR] \src\main\java\org\cactoos\text\AsciiLatin.java[77-77]: The String literal "�?" appears 5 times in this file; the first occurrence is on line 77 (AvoidDuplicateLiterals)
- [ERROR] \src\main\java\org\cactoos\text\AsciiLatin.java[58-58]: The String literal "�?" appears 5 times in this file; the first occurrence is on line 58 (AvoidDuplicateLiterals)
- [ERROR] \src\main\java\org\cactoos\text\AsciiLatin.java[70-70]: The String literal "�?" appears 3 times in this file; the first occurrence is on line 70 (AvoidDuplicateLiterals)
- [ERROR] \src\main\java\org\cactoos\text\AsciiLatin.java[62-62]: The String literal "�?" appears 2 times in this file; the first occurrence is on line 62 (AvoidDuplicateLiterals)
- [ERROR] \src\main\java\org\cactoos\text\AsciiLatin.java[125-125]: The String literal "�?" appears 2 times in this file; the first occurrence is on line 125 (AvoidDuplicateLiterals)
- [ERROR] \src\main\java\org\cactoos\text\AsciiLatin.java[59-59]: The String literal "�?" appears 3 times in this file; the first occurrence is on line 59 (AvoidDuplicateLiterals)
- [ERROR] \src\main\java\org\cactoos\text\AsciiNonLatin.java[69-69]: The String literal "�?" appears 2 times in this file; the first occurrence is on line 69 (AvoidDuplicateLiterals)

**Comment by ixmanuel** [comment]
Changing from master to AsciiText#235

**Comment by 0crat** [comment]
I can't do it for technical reasons, I'm very sorry. If you don't know what to do, submit this error as a ticket [here](https://github.com/zerocracy/datum):

```
1.0-SNAPSHOT ${buildNumber} 2017-06-29T17:45:24
java.io.IOException: java.lang.IllegalStateException: com.zerocracy.jstk.SoftException: Job `gh:yegor256/cactoos#236` was not in scope
	at org.cactoos.func.IoCheckedFunc.apply(IoCheckedFunc.java:74)
	at com.zerocracy.farm.SmartFarm.lambda$null$5(SmartFarm.java:142)
	at com.zerocracy.farm.StkSafe.process(StkSafe.java:73)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:91)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:67)
	at com.zerocracy.farm.reactive.Spinner.process(Spinner.java:120)
	at com.zerocracy.farm.reactive.Spinner.next(Spinner.java:105)
	at com.zerocracy.farm.reactive.Spinner.process(Spinner.java:81)
	at com.zerocracy.farm.reactive.Spinner.run(Spinner.java:67)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExec...
```

**Pull request closed without merging**